File: threads/src/main/java/im/threads/internal/transport/threadsGate/Action.java
package im.threads.internal.transport.threadsGate;

import com.google.gson.annotations.SerializedName;

public enum Action {
    @SerializedName("registerDevice")
    REGISTER_DEVICE,
    @SerializedName("sendMessage")
    SEND_MESSAGE,
    @SerializedName("getMessages")
    GET_MESSAGES,
    @SerializedName("getStatuses")
    GET_STATUSES,
    @SerializedName("updateStatuses")
    UPDATE_STATUSES;
}

File: src/main/java/im/threads/business/transport/threadsGate/Action.kt
package im.threads.business.transport.threadsGate

import com.google.gson.annotations.SerializedName

enum class Action {
    @SerializedName("registerDevice")
    REGISTER_DEVICE,

    @SerializedName("sendMessage")
    SEND_MESSAGE,

    @SerializedName("getMessages")
    GET_MESSAGES,

    @SerializedName("getStatuses")
    GET_STATUSES,

    @SerializedName("updateStatuses")
    UPDATE_STATUSES,

    UNDEFINED
}
